UnitedIncome/components
# Itemization widget update

### Description
Updates the itemization widget to support a custom total label

### Testing Instructions
1. Check out the story for the Itemization Widget
1. Use the totalLabel knob

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.
![image](https://user-images.githubusercontent.com/3478163/70252329-42df3100-174f-11ea-884a-cc352046890a.png)


### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4216)
